generators/server/templates/src/main/java/package/web/rest/errors/_DefaultException.java

```diff
@@ -0,0 +1,32 @@
+<%#
+ Copyright 2013-2017 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see http://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+package <%=packageName%>.web.rest.errors;
+
+import org.zalando.problem.AbstractThrowableProblem;
+import org.zalando.problem.Status;
+
+/**
+ * Generic exception with a message, that returns an Internal Server Error code.
+ */
+public class DefaultException extends AbstractThrowableProblem {
```

Comment:
In zalando there is already a DefaultProblem class so i'm wondering if it could be confusing to have DefaultProblem(zalanda) and DefaultException(jhipster) implementing both Exception and Problem ?

Reply:
Oh yes, let's rename this... GenericException?

Reply:
Agreed ! The best would be to `throw ProblemBuilder.setTitle('whatever').build()`

Reply:
No I can't use `ProblemBuilder` as `withTitle` is not a static method.
Besides using a `GenericException` is more concise

Reply:
Maybe `SimpleMessageException` is better. At least there is a meaning in the name

Reply:
Oups ! It's Problem.builder().withTitle... !

Reply:
Oh OK! Still I like `SimpleMessageException`, it's smaller - is it fine with you?

Reply:
Yes. And I just saw you also change the `type` so a custom exception is OK.
Actually I would vote for `InternalServerException` since that's the status used.

Reply:
OK let's go for a `InternalServerErrorException`

Reply:
:+1: